js/templates/pageNav.html

```diff
@@ -16,30 +16,30 @@
       </div>
       <div>
         <div class="flexVCent">
-          <a class="iconBtn ion-chevron-left js-navBack"></a>
+          <a class="iconBtn margLSm ion-chevron-left js-navBack"></a>
           <a class="iconBtn ion-chevron-right js-navFwd"></a>
-          <a class="iconBtn ion-refresh js-navReload"></a>
+          <a class="iconBtn ion-refresh js-navReload margRSm"></a>
         </div>
       </div>
-      <div class="rowDivV clrBrBk"></div>
+      <div class="rowDivV margR clrBrBk"></div>
       <div class="pageNavCenter">
         <div class="flexVCent gutterHSm">
           <div class="searchWrapper">
-            <input type="text" class="js-addressBar flexExpand addressBar clrBr4"
+            <input type="text" class="js-addressBar flexExpand addressBar clrSh2 clrBr4"
                    placeholder="<%= ob.polyT('addressBarPlaceholder') %>" value="<%= ob.addressBarText %>" />
           </div>
           <% if (ob.testnet) { %>
             <div id="testnetFlag" class="btn barBtn toolTip clrP clrBr" data-tip="<%= ob.polyT('testnetTooltip') %>">
               <%= ob.polyT('testnet') %>
             </div>
           <% } %>
-          <a class="btn barBtn clrP clrBr ion-navicon-round"></a>
+          <a class="bookmarkPage btn barBtn clrP clrBr clrSh2 tx4 ion-android-add"></a>
         </div>
       </div>
       <div class="rowDivV clrBrBk"></div>
       <div>
         <div class="flexVCent box">
-          <a class="iconBtn ion-eye"></a>
+          <a class="iconBtn margLSm ion-eye"></a>
```

Comment:
If you make a class for left and right padding around sets of icons, it could work here too. I think that would make sense as a pattern we'd want to repeat.

To make it more generic, it could also be something like an "iconRow" class that wraps any horizontal set of icons, and puts padding on the left and right side (though this one is missing the padding on the right side?).